src/components/ProtectedRoute.js

```diff
@@ -46,11 +48,51 @@ class ProtectedRoute extends Component {
             )
           )}
         />
+        <Divider section />
+        <div style={{ textAlign: 'center', margin: '10px 10% 10px 10%', paddingBottom: '20px' }}>
+          <p>
+            <FormattedMessage
+              id="app.protected_route.conduct_message"
+              description="Rovercode Code of Conduct"
+              defaultMessage="All users must follow the "
+            />
+            <a href="https://docs.rovercode.com/conduct/code-of-conduct">
+              <FormattedMessage
+                id="app.protected_route.conduct_link"
+                description="Rovercode Code of Conduct"
+                defaultMessage="Rovercode Code of Conduct."
+              />
+            </a>
+            <FormattedMessage
+              id="app.protected_route.inappropriate_message"
+              description="Instructions for what a student should do if they experience harassment"
+              defaultMessage=" If you see in appropriate behavior or feel you are being harassed, please stop using Rovercode and tell your teacher"
+            />
+            <FormattedMessage
+              id="app.protected_route.teachers_message"
+              description="Teacher conduct reporting link"
```

Comment:
This one is similar with something like `Description and link to inform the teacher of where to report issues`